samtranslator/translator/translator.py

```diff
@@ -31,6 +31,36 @@ def __init__(self, managed_policy_map, sam_parser, plugins=None):
         self.plugins = plugins
         self.sam_parser = sam_parser
 
+    def _get_function_names(self, resource_dict, parameter_values):
```

Comment:
Is it not possible to use the intrinsic resolver? This looks like a lot of duplicated effort.

Intrinsic resolver does additional things, like resolving "!Sub ${parameter}-name", which this doesn't appear to handle.

Reply:
intrinsics resolver is not being called from this file. I will check and get back if i can call intrinsics resolver here.